packages/cli/src/commands/init-flow.ts

```diff
@@ -62,7 +62,7 @@ export const initFlow = async (
 
   if (shouldInstallDeps === true) {
     const spinner = ora().start();
-    spinner.text = "Installing dependencies";
+    spinner.text = "Installing Dependencies";
```

Comment:
Why "Dependencies"? Not a title or heading, just a sentence.

Reply:
Changed it, i was using `D` in the same file. Now changed in both the places 👍